lib/fathead/mdn_css/parse.pl

```diff
@@ -160,29 +151,21 @@
           make_external_links( $link, $external_lis_collection );
     }
     $external_links ||= '';
-
-    push @entries,
-      make_article( $title_clean, $description, $link, $external_links );
-
-    # Check for CSS Functions
-    # e.g. "not()"
-    # Replace "()" with "function" in title
-    # e.g. "not()" - > "not function"
-    if ( $title_clean =~ m/\(\)$/ ) {
-        say "Found Function: $title_clean";
-        my $temp = $title_clean;
-        $temp =~ s/\(\)$/ function/;
-        push @entries, make_redirect( $temp, $title );
-        $temp = $title_clean;
-        $temp =~ s/\(\)$//;
-        push @entries, make_redirect( $temp, $title );
-    }
-
-    write_to_file(@entries);
+    make_and_write_article( $title, $description, $link, $external_links );
 }
 
 # PRIVATE FUNCTIONS
 
+sub clean_code {
+    my ($code) = @_;
+    my $hs = HTML::Strip->new( emit_spaces => 0 );
+    $code = $hs->parse($code);
+    $code =~ tr/ / /s;
+    $code =~ s/\r?\n/\\n/g;
```

Comment:
@hchienjo to fix `deg` we might need to do something like `tr/\n/\n/s` to remove duplicate newlines?